criu/Makefile.crtools

```diff
@@ -40,6 +40,8 @@ obj-y			+= log.o
 obj-y			+= lsm.o
 obj-y			+= mem.o
 obj-y			+= memfd.o
+obj-y                   += secretmem.o
+obj-y                   += memfd-secret.o
```

Comment:
Thanks @Snorch :) just fixed the lint.

Re: other CI tests: most of the CI tests are failing because, they don't have access to `memfd_secret` sys call. To enable `memfd_secret` sys call, we have to set kernel boot parameter `secretmem.enable=1`. How can I do this in CI?

Reply:
That is a good question:

First, you should detect syscall availability in kerndat and skip zdtm tests for it and skip criu using it in case it is not available.

Second, we need to have test in CI with secretmem.enable=1, but I'm not sure how to do it, maybe someone more experienced with CI knows if we can control kernel boot options? @checkpoint-restore/maintainers

Reply:
thanks @Snorch

Reply:
I have implemented the `.desc` [file](https://github.com/checkpoint-restore/criu/blob/4992514cbff0748622ca0abb4fc6b2ecc638b948/test/zdtm/static/memfd-secret00.desc) to skip zdtm tests if `memfd_secret` [feature](https://github.com/checkpoint-restore/criu/blob/4992514cbff0748622ca0abb4fc6b2ecc638b948/criu/cr-check.c#L1635) isn't available.

Also, to fix the failing CI tests due to the unavailability of `memfd_secret` syscall, I tried modifying the `FEATURES_LIST` in [Makefile.config](https://github.com/checkpoint-restore/criu/blob/4992514cbff0748622ca0abb4fc6b2ecc638b948/Makefile.config#L82). And updated the [`scripts/feature-tests.mak`](https://github.com/checkpoint-restore/criu/blob/4992514cbff0748622ca0abb4fc6b2ecc638b948/scripts/feature-tests.mak#L175). Then iteratively (looking at the failing CIs), included conditional compilation macros around the `memfd_secret` related code. For example see [`kerndat_get_secretmem_dev()`](https://github.com/checkpoint-restore/criu/blob/4992514cbff0748622ca0abb4fc6b2ecc638b948/criu/kerndat.c#L261).

I'm not this is the right approach to do this. But, these changes fix the important CI tests such as `ci/circleci: test-local-gcc` and ` ci/circleci: test-local-clang`. What do you think @Snorch :)

Reply:
understood, thanks @rppt :)